File: C++17/백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.cc
#include <iostream>
using namespace std;

int main (void) {
    int N, A, B;
    cin >> N >> A >> B;
    
    if (A > B) {
        cout << "Subway" << endl;
    } else if (A < B) {
        cout << "Bus" << endl;
    } else {
        cout << "Anything" << endl;
    }
    
    return 0;
}

File: Java/백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        long N = sc.nextLong();
        long A = sc.nextLong();
        long B = sc.nextLong();
        
        if (A > B) {
            System.out.println("Subway");
        } else if (A < B) {
            System.out.println("Bus");
        } else {
            System.out.println("Anything");
        }
        
        sc.close();
    }
}